app/models/farmers/farmer.py

```
@@ -0,0 +1,23 @@
+from sqlalchemy import Boolean, Column, Float, ForeignKey, String, Text
```

Ruff (0.12.2)

`sqlalchemy.Column` imported but unused

Remove unused import

(F401)